sig-autoscaling/README.md

```diff
@@ -13,7 +13,7 @@ Covers development and maintenance of components for automated scaling in Kubern
 The [charter](charter.md) defines the scope and governance of the Autoscaling Special Interest Group.
 
 ## Meetings
-* Regular SIG Meeting: [Mondays at 14:00 UTC](https://zoom.us/j/944410904) (biweekly/triweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=14:00&tz=UTC).
+* Regular SIG Meeting: [Mondays at 14:00 Poland](https://zoom.us/j/944410904) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=14:00&tz=Poland).
```

Comment:
It is 16:00 in Poland. And maybe let's use European (CEST/CET) time zone instead of Poland.

Reply:
Agh, silly mistake, spent so long trying to find a timezone that worked I forgot to actually update the time.

The problem with CEST/CET is that the link to convert to the local timezone then no longer works, and as far as I'm aware there's no simple timezone to use that gives whatever the current prevalent timezone in Europe is.

Reply:
Poland does work with the time converter and it should follow DST.
I'm okay with approving as is just to prevent having to update again the next time a DST change is needed.

Reply:
Thanks for checking the conversion, I was never able to figure out how to full verify that.